tests/CssOrXpathExpressionTranslatorTest.php

```diff
@@ -97,26 +97,20 @@ public function testQueries(string $input, string $expect): void
     {
         $output = CssOrXpathExpressionTranslator::getTranslator()
             ->convertToXpath($input);
-        static::assertEquals($expect, $output);
+        self::assertEquals($expect, $output);
```

Comment:
```suggestion
       $this->assertEquals($expect, $output);
```

Reply:
self::assertSame(), but $this->assertEquals()? Why?

Reply:
To have the same syntax across all tests

Reply:
![image](https://user-images.githubusercontent.com/23000680/116056658-b3f5f000-a686-11eb-92fa-8f7a10a63269.png)

Reply:
Tests mostly use static methods. expectException() is exception

Reply:
You wrote a line `$this->expectException(\InvalidArgumentException::class);` - so why you are writing one statement in one style and other in another style. 


Please make it consistent. I'm good with `self::assert*`

Reply:
![image](https://user-images.githubusercontent.com/23000680/116058711-c7a25600-a688-11eb-9c22-1d23577a2ea0.png)
I don't khow why it works in this way. But as long as I remember it was always like this.

Reply:
Maybe your inspection is misconfigured?

Reply:
You're right. It's really just an inspection. And which approach is used does not matter: https://phpunit.de/manual/6.5/en/appendixes.assertions.html#appendixes.assertions.static-vs-non-static-usage-of-assertion-methods